### PR TITLE
feat(docs): move versioned_sidebars to static-assets

### PR DESCRIPTION
**Pull request by yoonhyejin**
## Checklist

- [ ] The PR conforms to DataHub's [Contributing Guideline](https://github.com/datahub-project/datahub/blob/master/docs/CONTRIBUTING.md) (particularly [Commit Message Format](https://github.com/datahub-project/datahub/blob/master/docs/CONTRIBUTING.md#commit-message-format))
- [ ] Links to related issues (if applicable)
- [ ] Tests for the changes have been added/updated (if applicable)
- [ ] Docs related to the changes have been added/updated (if applicable). If a new feature has been added a Usage Guide has been added for the same.
- [ ] For any breaking change/potential downtime/deprecation/big changes an entry has been made in [Updating DataHub](https://github.com/datahub-project/datahub/blob/master/docs/how/updating-datahub.md)

This PR should be merged first. https://github.com/datahub-project/static-assets/pull/50

### CODE REVIEW

File: docs-website/download_historical_versions.py [hunk 1]
```diff
@@ -16,30 +16,36 @@ def download_file(url, destination):
                 f.write(chunk)
 
 
-def fetch_tar_urls(repo_url, folder_path):
-    api_url = f"{repo_url}/contents/{folder_path}"
+def fetch_urls(repo_url: str, folder_path: str, file_format: str):
+    api_url = f"{repo_url}/contents/{folder_path}?ref=versioned_sidebar"
```
Comment (hsheth2) [inline docs-website/download_historical_versions.py:20]: revert this change

**Review by hsheth2 — COMMENTED** [review]
one change needed but lgtm

**Review by hsheth2 — APPROVED** [review]
one change needed but lgtm

**Review by hsheth2 — APPROVED** [review]
one change needed but lgtm

File: docs-website/download_historical_versions.py (hunk 1 above)
Comment (yoonhyejin) [reply docs-website/download_historical_versions.py]: nice catch -- thx!
